WordPress/Classes/ViewRelated/NUX/WPStyleGuide+Login.swift

```diff
@@ -93,7 +93,26 @@ extension WPStyleGuide {
         return UIFont.systemFont(ofSize: maxAllowedFontSize, weight: .medium)
     }
 
-    // MARK: - Google Signin Button Methods
+    /// Creates a attributed string with one underlined section that's surrounded by underscores
+    /// Such as "this _is_ underlined" would under the "is"
+    ///
+    class func underline(formattedString baseString: String, color: UIColor? = nil, underlineColor: UIColor? = nil) -> NSAttributedString {
```

Comment:
Since the color parameters aren't really taken from WPStyleGuide, this one would probably make more sense as a `NSAttributedString` extension, perhaps?